packages/dapp/src/const/tabsMap.ts

```diff
@@ -0,0 +1,15 @@
+import { WidgetSubvariant } from '@lifi/widget/types';
+
+interface TabsMapType {
+  [key: string]: {
+    index: number;
+    value: WidgetSubvariant | 'buy';
+  };
+}
+
+export const TabsMap: TabsMapType = {
+  Exchange: { index: 0, value: 'default' },
+  Gas: { index: 1, value: 'refuel' },
+  Refuel: { index: 1, value: 'refuel' },
```

Comment:
Why do we have gas and refuel? Maybe we should select one and proceed with it 🙂

Reply:
Makes sense! Good find, done